packages/rspack/src/exports.ts

```diff
@@ -261,6 +261,11 @@ export type {
 	SwcLoaderTsParserConfig
 } from "./builtin-loader/swc/index";
 
+import * as lightningcss from "./builtin-loader/lightningcss/index";
+
+export { type LoaderOptions as LightningcssLoaderOptions } from "./builtin-loader/lightningcss/index";
+export { lightningcss };
```

Comment:
This still needs further discussion, @chenjiahan any suggestions?

Reply:
This is the first `rspack.xxx` that not same with webpack, even there are some plugins specific for rspack, they are postfixed by `RspackPlugin`, we should at least try our best to avoid the consistency here 

And my suggestion is:

```js
{
  loader: "builtin:lightningcss-loader"
  option: {
    exclude: { // add a layer to transform the user config into the raw config, instead of exposing the raw config and utility function directly 
      nesting: true
    }
  }
}
```

Reply:
LGTM, we should avoid export { ligntningcss } from Rspack core.